### PR TITLE
Update household max size chart by unit type

### PR DESCRIPTION
**Pull request by software-project**

### CODE REVIEW

**Comment by software-project** [comment]
@bk3c I've addressed most of @slowbot issues. Will just add <strong> thing tomorrow. But I can't make new field on units get sent to the frontend. I've tried regenerating client but it didn't help. Could you or @jaredcwhite help?

**Comment by software-project** [comment]
@bk3c @kathyccheng @slowbot first column is strong now and subheader is fixed now. So should be ready to go.

File: backend/core/src/entity/unit.entity.ts [hunk 1]
```diff
@@ -149,6 +149,11 @@ class Unit extends BaseEntity {
     onUpdate: "CASCADE",
   })
   listing: Listing
+
+  @Column({ type: "boolean", nullable: true })
+  @Expose()
+  @IsOptional()
+  bmrProgramChart?: boolean
```
Comment (pbn4) [inline backend/core/src/entity/unit.entity.ts:156]: Please add `@IsBoolean` under IsOptional and change `barProgramChart?: boolean` to `bmrProgramChart: boolean | null`

File: services/listings/src/lib/unit_transformations.ts [hunk 2]
```diff
@@ -32,16 +32,19 @@ const minMaxInCurrency = (minMax: MinMax): MinMaxCurrency => {
   return { min: usd.format(minMax.min), max: usd.format(minMax.max) }
 }
 
+const bmrHearders = ["Studio", "1 BR", "2 BR", "3 BR", "4 BR"]
```
Comment (pbn4) [inline services/listings/src/lib/unit_transformations.ts:35]: Headers*

**Review by pbn4 — APPROVED** [review]

**Review by bk3c — APPROVED** [review]
Ok -- I'm pretty sure that we're going to want to re-think which of these rules are on units and which are on listings in the future, but this works ok for me in the short-term, so let's go with it for now.

**Pull request merged**